frontend/src/components/worksheets/items/TableItem/BundleRow.js

```diff
@@ -481,11 +481,15 @@ const styles = (theme) => ({
     root: {
         verticalAlign: 'middle !important',
         border: 'none !important',
+        wordWrap: 'break-word',
```

Comment:
These make text in bundle rows wrap and extend vertically when the text is longer than the section length